fhir-config/src/main/java/com/ibm/fhir/config/FHIRConfiguration.java

```diff
@@ -35,6 +35,7 @@
     public static final String PROPERTY_SERVER_REGISTRY_RESOURCE_PROVIDER_ENABLED = "fhirServer/core/serverRegistryResourceProviderEnabled";
     public static final String PROPERTY_CAPABILITY_STATEMENT_CACHE = "fhirServer/core/capabilityStatementCacheTimeout";
     public static final String PROPERTY_EXTENDED_CODEABLE_CONCEPT_VALIDATION = "fhirServer/core/extendedCodeableConceptValidation";
+    public static final String PROPERTY_DISALLOWED_OPERATIONS = "fhirServer/core/disallowedOperations";
```

Comment:
My preference would be for "DISABLED_OPERATIONS" instead of "DISALLOWED_OPERATIONS".